thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/bao/SubscriptionGroupManagerImpl.java

```diff
@@ -16,15 +16,36 @@
 import ai.startree.thirdeye.datalayer.dao.GenericPojoDao;
 import ai.startree.thirdeye.spi.datalayer.bao.SubscriptionGroupManager;
 import ai.startree.thirdeye.spi.datalayer.dto.SubscriptionGroupDTO;
+import com.codahale.metrics.CachedGauge;
+import com.codahale.metrics.MetricRegistry;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
+import java.util.concurrent.TimeUnit;
 
 @Singleton
 public class SubscriptionGroupManagerImpl extends
     AbstractManagerImpl<SubscriptionGroupDTO> implements SubscriptionGroupManager {
 
   @Inject
-  public SubscriptionGroupManagerImpl(GenericPojoDao genericPojoDao) {
+  public SubscriptionGroupManagerImpl(final GenericPojoDao genericPojoDao,
+      final MetricRegistry metricRegistry) {
     super(SubscriptionGroupDTO.class, genericPojoDao);
+    metricRegistry.register("subscriptionsCountTotal",
```

Comment:
This name is confusing. If you ask someone from the team what `subscriptionsCountTotal` is, what do you think they would answer?

Reply:
Trust me I had hard time finalizing a name for this metric. Other contender was `notificationFlowCountTotal`. Picked the existing one as we already have a well defined entity `SubscriptionGroup` so atleast no one will confuse the count with `SubscriptionGroup` entity count and at the same time make is obvious that it's related to it somehow.
Any good suggestion is welcomed!